src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -861,6 +856,11 @@
       "type": "object",
       "properties": {
 
+        "analysisTarget": {
+          "description": "Identifies the file that the analysis tool was instructed to scan. This need not be the same as the file where the result actually occurred.",
+          "$ref": "#/definitions/fileLocation"
+        },
+
```

Comment:
Let's move this down to right above the `locations` property, since they're related.